tests/e2e/registry/test_registry_image_push_pull.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import pytest
+from ocs_ci.framework.testlib import tier1, E2ETest, ignore_leftovers
+from ocs_ci.ocs import ocp, registry, constants
+from ocs_ci.framework import config
+
+logger = logging.getLogger(__name__)
+
+
+class TestRegistryImagePullPush(E2ETest):
+    """
+    Test to check Image push and pull worked with registry backed by OCS
+    """
+
+    @tier1
+    @ignore_leftovers
+    @pytest.mark.polarion_id("OCS-1080")
+    def test_registry_image_pull_push(self):
+        """
+        Test case to validate registry image pull and push with OCS backend
+        """
+        image_url = 'docker.io/library/busybox'
+
+        # Get openshift registry route and certificate access
+        registry.enable_route_and_create_ca_for_registry_access()
+
+        # Add roles to user so that user can perform image pull and push to registry
```

Comment:
Roles are not removed from user at the end of the test case so there is a change of test environment for all other test cases triggered after this one. Although, I am not sure if it  should be done in this PR as there would need to be added methods `get_user_roles` and `delete_user_roles` to `registry` (or some new module concerned with user policy).

Reply:
@fbalak There is cmd to remove user-roles, let me add a function or fixture. But didn't faced any issues with respect to running other tests after changing roles. 
Modifying the user roles allows to perform operations like image-push and pull to the registry namespace.

Reply:
Updated code with removing the user roles at the end.